src/utils.js

```diff
@@ -24,29 +24,27 @@ var getCommentFormatter = function (file) {
   var extension = file.relative.split('.').pop(),
     fileContents =  file.contents.toString(),
     newline =  detectNewline.graceful(fileContents || ''),
+    newlinePre =  newline,
     commentFormatter = function(url) {
       return '';
     };
 
   if (file.sourceMap.preExistingComment){
     debug(function() { return 'preExistingComment commentFormatter'; });
-    commentFormatter = function(url) {
```

Comment:
For current code `preExistingComment` prevents returning a string that starts with a line-feed.  The existing code was fine for JS but not for CSS (`//` isn't a comment in CSS).  We have a test for JS with a pre-existing comment but not for CSS with pre-existing comment.  Honestly I'm not sure how to create a CSS file with a valid pre-existing source-map comment.

I'm not sure that it's very important to avoid adding a blank line between the source and the new comment, but the `newlinePre` variable is used to avoid that blank line and maintain current behavior.